### PR TITLE
Update adc_driver.cc

### PR DESCRIPTION
**Pull request by johnw188**

### CODE REVIEW

File: adc_driver.cc [hunk 1]
```diff
@@ -100,6 +99,7 @@ unsigned int adc_driver::read_once (unsigned char channel)
  *  \brief Overloaded insertion operator
  *  @param serial A reference to the serial-type object to which to print
  *  @param my_adc A reference to the adc_driver object to display data for
+ * test
```
Comment (johnw188) [inline adc_driver.cc]: Why test?

File: adc_driver.cc [hunk 2]
```diff
@@ -31,7 +31,6 @@
 /** This union holds two bytes, making them accessable as both a two char array and
  *  a single 16 bit word
  *  \brief Union to hold the result of an A/D conversion
- * lol comments
```
Comment (johnw188) [inline adc_driver.cc:34]: I liked this

**Review by johnw188 — COMMENTED** [review]
Fix this

File: adc_driver.cc (hunk 2 above)
Comment (BobElDevil) [inline adc_driver.cc:34]: good thing you deleted this, it isn't lined up with the other comments

File: adc_driver.cc (hunk 1 above)
Comment (BobElDevil) [inline adc_driver.cc]: this one needs an extra space

**Review by BobElDevil — CHANGES_REQUESTED** [review]
Needs work

File: adc_driver.cc (hunk 2 above)
Comment (johnw188) [reply adc_driver.cc:34]: You're wrong

File: adc_driver.cc (hunk 1 above)
Comment (johnw188) [reply adc_driver.cc]: Ok

**Review by BobElDevil — APPROVED** [review]

File: adc_driver.cc [hunk 3]
```diff
@@ -31,7 +31,6 @@
 /** This union holds two bytes, making them accessable as both a two char array and
  *  a single 16 bit word
  *  \brief Union to hold the result of an A/D conversion
```
Comment (BobElDevil) [inline adc_driver.cc:33]: Showing this to travis

File: adc_driver.cc [hunk 4]
```diff
@@ -100,6 +99,7 @@ unsigned int adc_driver::read_once (unsigned char channel)
  *  \brief Overloaded insertion operator
  *  @param serial A reference to the serial-type object to which to print
  *  @param my_adc A reference to the adc_driver object to display data for
```
Comment (BobElDevil) [inline adc_driver.cc:101]: and here

**Review by BobElDevil — COMMENTED** [review]
This was a review

**Pull request merged**